tool/e2e-tests/prescribe/sendPrescriptionsFromTestPack.ts

```diff
@@ -3,7 +3,7 @@ import {sendBulkPrescriptionUserJourney} from "../helpers"
 import {loadClinicalTestPack1Examples} from "../test-packs/test-packs"
 
 describe("firefox", () => {
-  test("can send prescriptions from test pack 1", async () => {
-    await sendBulkPrescriptionUserJourney(driver, loadClinicalTestPack1Examples, 5)
+  test("can send prescriptions from clinical test pack 1", async () => {
+    await sendBulkPrescriptionUserJourney(driver, loadClinicalTestPack1Examples, 25)
```

Comment:
why 25? the example xlsx used has 214 entries

Reply:
The signature requests/responses are made in batches of 25, so this test waits for the first 25 to come back successfully (25 ticks on screen) and then moves on